benchmarks/dynamo/huggingface.py

```diff
@@ -437,6 +437,7 @@ def forward_pass(self, mod, inputs, collect_outputs=True):
     def forward_and_backward_pass(self, mod, inputs, collect_outputs=True):
         cloned_inputs = clone_inputs(inputs)
         mod.zero_grad(True)
+        self.optimizer_zero_grad()
```

Comment:
you dont need `mode.zero_grad(True)` and `self.optimizer_zero_grad()`. Only one of them is sufficient I believe.

Reply:
Cool, will keep optimizer.zero_grad as it seems to be more popular.